publishable/database/migrations/2016_01_01_000000_add_voyager_user_fields.php

```diff
@@ -10,8 +10,9 @@ class AddVoyagerUserFields extends Migration
     public function up()
     {
         Schema::table('users', function ($table) {
-            $table->string('avatar')->nullable()->after('email');
             $table->integer('role_id')->nullable()->after('id');
+            $table->string('avatar')->nullable()->after('email');
+            $table->mediumText('bio')->nullable()->after('avatar');
```

Comment:
The new `bio` field should be turned into a new migration since `v0.11` already is out.